packages/common/src/schemas/upload/uploadFormSchema.ts

```diff
@@ -25,6 +25,8 @@ const messages = {
   }
 }
 
+const MAX_DESCRIPTION_LENGTH = 2500
```

Comment:
Is there a way to get this value from the backend through the SDK to then being used here? All good to have it like this just wondering if there's a way to have a single source of truth

Reply:
yeah would be nice but would have to have a whole endpoint and everything, don't think it's worth it for this tbh

Reply:
Ah right the autogenerating is just for types